frontend/server/src/DAO/CoderOfTheMonth.php

```diff
@@ -142,7 +142,7 @@ final public static function getCodersOfTheMonthFromSchool(
 
     /**
      * Get all coder of the months based on month
-     * @return list<array{classname: string, country_id: string, email: null|string, ranking: int, time: string, user_id: int, username: string}>
+     * @return list<array{classname: string, country_id: string, email: null|string, problems_solved: int, rank?: int, score: float, time: string, user_id?: int, username: string}>
```

Comment:
Esta función no cambió, así que podrás regresar el tipo como estaba antes?
```suggestion
     * @return list<array{classname: string, country_id: string, email: null|string, ranking: int, time: string, user_id: int, username: string}>
```